snap/hooks/post-refresh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+DICPATH=$SNAP_COMMON/snap-hunspell
+if [ ! -d $SNAP_COMMON/host-hunspell ]; then
+  echo "No host-hunspell, skipping"
+  exit 0
+fi;
+
+if [ -d $DICPATH ]; then
+  # Cleanup on each refresh, ensure we have uptodate content
+  find $DICPATH -type l -name "*.dic" -or -name "*.aff" | xargs rm
+else
+  mkdir -p $DICPATH
+fi
```

Comment:
It was breaking when I was performing the install on top of a stable firefox snap

Reply:
Probably because `$DICPATH` didn't exist, but the `-f` option to `rm` would ignore a non-existing directory. That's bike shedding anyway, this approach works, so let's leave it like this.